src/test/java/com/hubspot/jinjava/lib/tag/FromTagTest.java

```diff
@@ -54,6 +56,13 @@ public void importedContextExposesVars() {
         .contains("wrap-padding: padding-left:42px;padding-right:42px");
   }
 
+  @Test
+  public void importedCycleDected() {
```

Comment:
Is it possible for A to import from B which imports from C which imports from A? If so can we have a test to make sure it catches this cycle as well?